config/samples/csharp_example_loadtest.yaml

```diff
@@ -24,10 +24,13 @@ spec:
     name: '0'
     run:
       args:
-      - exec
-      - qps_worker/Grpc.IntegrationTesting.QpsWorker.dll
+      - -c
+      - |
+        timeout --kill-after=$${KILL_AFTER} $${POD_TIMEOUT} dotnet exec
+        qps_worker/Grpc.IntegrationTesting.QpsWorker.dll
+        --driver_port=$${DRIVER_PORT}
```

Comment:
qq: why the double `$$`? (I might be missing something).

Reply:
Loadtest_template.py uses these examples to generate loadtest template, see [grpc/tools/run_tests/performance/templates/loadtest_template_basic_all_languages.yaml](https://github.com/grpc/grpc/blob/master/tools/run_tests/performance/templates/loadtest_template_basic_all_languages.yaml) and add some injection field, when the Loadtest_config.py takes the generated template to produce runnable loadtest configurations, it will require all fields with $ to be substitute with a actual values such as  ${client_pool}. KILL_AFTER, POD_TIMEOUT and DRIVER_PORT are used inside the container, we do not want to substitute actual values for them when we generating the configuration file, $$ is to escape. There could be other ways of doing it, open to modify if there are concerns.

Reply:
Moving forward we will not use $$, updating the pr.